packages/manager/src/components/SelectRegionPanel/SelectRegionPanel.tsx

```diff
@@ -62,16 +67,27 @@ const SelectRegionPanel: React.FC<Props & WithStyles<ClassNames>> = (props) => {
     selectedID,
   } = props;
 
+  const location = useLocation();
+
   if (props.regions.length === 0) {
     return null;
   }
 
+  const params = getParamsFromUrl(location.search);
+  const showCrossDataCenterCloneWarning =
+    /clone/i.test(params.type) && params.regionID !== selectedID;
```

Comment:
This is very minor, but it seems like `selectedID` is sometimes an empty string when `params.regionID` is `undefined`, resulting in the Notice showing when is does not need to




https://user-images.githubusercontent.com/115251059/229572465-e02b324e-43ae-4468-9a63-740163f95a8b.mov


I found that adding a check for `selectedID` fixes this.
 ```suggestion
     /clone/i.test(params.type) && selectedID && params.regionID !== selectedID;
```